adapter-pinecone/src/main/java/io/nosqlbench/adapter/pinecone/PineconeDriverAdapter.java

```diff
@@ -0,0 +1,44 @@
+/*
+ * Copyright (c) 2022 nosqlbench
```

Comment:
2023 update to license header.  Appears it may be throughout.

Reply:
There are instructions for automating this in IntelliJ on the dev guide section.

Reply:
Updated in files and updated IntelliJ to automate process in the future.